packages/data-point/package.json

```diff
@@ -52,7 +52,7 @@
   },
   "files": [
     "LICENSE",
-    "index.js",
+    "*.js",
     "lib/"
```

Comment:
is `"lib/"` still necessary now that we're also doing `".js"`?

Reply:
actually I should rever this change